brokers/unified/broker_test.go

```diff
@@ -1066,7 +1066,7 @@ func TestBroker_getPluginMetas(t *testing.T) {
 				defaultRegistry: defaultRegistry,
 			},
 			want: want{
-				errRegexp: regexp.MustCompile("failed to fetch plugin meta.yaml for plugin .* from registry .*"),
+				errRegexp: regexp.MustCompile("failed to fetch plugin meta.yaml from URL .*"),
```

Comment:
Please consider adding test case for new use-case with `reference` specified

Reply:
added